packages/react/src/components/Storage/File-Uploader/__tests__/FileUploader.test.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { FileUploader } from '..';
+
+describe('File Uploader', () => {
+  it('exists', async () => {
+    const comp = render(
+      <FileUploader
+        level="public"
+        acceptedFileTypes={['.png']}
+        variation="drop"
+      />
+    );
+
+    expect(comp.container).toBeTruthy();
```

Comment:
Same nit as above, would prefer snapshot test, or some extensive UI assertions

Reply:
OK, will add to future PR